extensions/amp-analytics/0.1/events.js

```diff
@@ -353,6 +353,12 @@ export class CustomBrowserEventTracker extends EventTracker {
 
   /** @override */
   dispose() {
+    const root = this.root.getRoot();
```

Comment:
Do you mean `getRootElement`?

Reply:
I used getRoot to keep the dispose function stylistically similar to the dispose function of the other trackers.

Reply:
Nevermind. Type of `root` threw me off. It's an abstract class.